Sources/vincenty/vincenty.swift

```diff
@@ -26,101 +40,146 @@ private let pi = Double.pi
 ///   - maxIter: maximal number of iterations
 ///   - a: first ellipsoid parameter in meters (defaults to WGS-84 parameter)
 ///   - f: second ellipsoid parameter in meters (defaults to WGS-84 parameter)
-/// 
-/// - Returns: distance between `x` and `y` in meters.
+///
+/// - Returns: distance between `x` and `y` in meters. initial True Track in degrees, and final True Track in degrees.
 ///
 /// - Throws:
-///   - `ConvergenceError.notConverged` if the distance computation does not converge within `maxIter` iterations.
-public func distance(_ x: (lat: Double, lon: Double),
+///   - `VincentyError.notConverged` if the distance computation does not converge within `maxIter` iterations.
+///   - `VincentyError.inputOutsideRange` if the input parameters are out of range
+public func vincentyCalculations(_ x: (lat: Double, lon: Double),
```

Comment:
And then add convenience function to call those functions in order to keep the API intact?
Like "directDistance"/"inverseDistance" etc?
Or add a parameter to select direct/inverse?
Sometimes breaking changes are good, when it brings clarity...

Reply:
I guess a parameter would not make too much sense since the parameters are different for the direct problem. So we should just leave room for another function name. Also keep in mind that the direct problem does not result in a distance, but a point instead. So `directDistance`, `inverseDistance` could be a bit misleading. Potentially having two functions `solveInverse` and `solveDirect` in a package called `vincenty` is the most concise function names.

Reply:
Hrrm. Right! Fixed.